### PR TITLE
Update license req for quote/sync #872

### PR DESCRIPTION
**Pull request by digitaldiina**
image

### CODE REVIEW

File: docs/en/quote/learn/admin/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ language: en
 SuperOffice Quote helps you to create, validate, and manage quotes and orders across the entire order lifecycle. It helps you get new or updated quotes to your customers faster and speed up the transition from a lead to a closed deal.
 
 > [!NOTE]
-> Quote management is a feature you get with the Sales premium license.
+> This feature requires a Sales Premium user plan and the Integration Services license.
```
Comment (SuperOfficeDevNet) [inline docs/en/quote/learn/admin/index.md:19]: I've heard of the Development Tools and Expander Tools licenses, never the Integration Services license... 
Is this a real license name? If it it, there should be some information about it that we can link to.

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Please review my comments about the license name.

File: docs/en/quote/learn/admin/index.md (hunk 1 above)
Comment (digitaldiina) [reply docs/en/quote/learn/admin/index.md:19]: It was for v8 (called Integration Server), but is now included with Development Tools (checked with CM).

I'll update the old mentions throughout. Brb

**Review by SuperOfficeDevNet — APPROVED** [review]
Better. Thanks for the corrections! 🏆

**Pull request merged**